mem_mgmt/mem_mgmt.c

```diff
@@ -158,7 +158,7 @@ nvm_status_t nvm_save_msp_seqflags(void)
 	status = NVM_unlock(NVM_SEQFLAG_ADDR, sizeof(msp_seqflags_t));
 	if((NVM_SUCCESS == status)||(NVM_WRITE_THRESHOLD_WARNING == status)){
 		status = NVM_write(NVM_SEQFLAG_ADDR,
-		                   (uint8_t*)&seqflags,
+		                   (uint32_t*)&seqflags,
```

Comment:
We talked about changing the cast in the function nvm_restore_msp_seqflags since the msp_seqflags union is 32 bits. I have  changed wrong part of the code here. Now changed back to uint8_t